src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -30,11 +30,28 @@ const IncidentSplitContainer = ({ FormComponent }) => {
   const dispatch = useDispatch();
   const [parentIncident, setParentIncident] = useState();
   const [directingDepartment, setDirectingDepartment] = useState([]);
+  const [note, setNote] = useState();
   const departments = useSelector(makeSelectDepartments);
   const directingDepartments = useSelector(makeSelectDirectingDepartments);
 
   const [subcategoryGroups, subcategoryOptions] = useSelector(makeSelectSubcategoriesGroupedByCategories);
 
+  const patchData = useMemo(() => {
+    if (!parentIncident) return {};
+
+    const shouldPatchDirectingDepartment =
+      !parentIncident.directing_departments ||
+      parentIncident.directing_departments.length !== directingDepartment.length ||
+      (!parentIncident.directing_departments.includes(department => department?.id === directingDepartment[0]?.id) &&
+        typeof directingDepartment[0]?.id !== 'undefined');
```

Comment:
Is this necessary, to check for `typeof` `undefined`?

Reply:
It was not - removed it